node-benchmark-compare.js

```diff
@@ -10,16 +10,39 @@ if (process.argv.length !== 3) {
   process.exit(1);
 }
 
+function parseCsvLine(data) {
+  let state = 0;
```

Comment:
Can you use named constants to make it clear what the states are?

Reply:
Yes